evm.md

```diff
@@ -1322,6 +1328,19 @@ The various `CALL*` (and other inter-contract control flow) operations will be d
          <program> _ => BYTES </program>
          <jumpDests> _ => #computeValidJumpDests(BYTES) </jumpDests>
 
+    syntax KItem ::= "#touchStorage" Account Int
+ // --------------------------------------------
+    rule <k> #touchStorage ACCT INDEX => . ... </k>
+         <touchedStorage> ... ACCT |-> (TS:Set => TS |Set SetItem(INDEX)) ... </touchedStorage>
+    rule <k> #touchStorage ACCT INDEX => . ... </k>
+         <touchedStorage> TS => TS[ACCT <- SetItem(INDEX)] </touchedStorage>
+      requires notBool ACCT in_keys(TS)
+
+    syntax Bool ::= inStorage ( Map , Account , Int ) [function]
+ // ------------------------------------------------------------
+    rule inStorage (TS, ACCT, _ )  => false requires notBool ACCT in_keys(TS)
+    rule inStorage (TS, ACCT, KEY) => KEY in {TS[ACCT]}:>Set requires ACCT in_keys(TS)
+
```

Comment:
This is only used for gas calculation, so move it to that section. Also prefix it with a hash `#inStorage`.